### PR TITLE
Completed react-props exercise - Shea

### PR DESCRIPTION
**Pull request by SheaPone**
Screenshot 2024-12-05 at 7 51 05 PM

### CODE REVIEW

File: react-props/my-app/src/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type Props = {
+  text: string;
+};
+
+export function Header(props: Props) {
```
Comment (gerardoh13) [inline react-props/my-app/src/Header.tsx]: This should be a destructured prop

File: react-props/my-app/src/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import './Image.css';
+
+type Props = {
+  src: string;
+};
+
+export function Image(props: Props) {
```
Comment (gerardoh13) [inline react-props/my-app/src/Image.tsx]: destructured prop

File: react-props/my-app/src/Caption.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+type Props = {
+  text: string;
+};
+
+export function Caption(props: Props) {
```
Comment (gerardoh13) [inline react-props/my-app/src/Caption.tsx]: destructured prop

File: react-props/my-app/src/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+type Props = {
+  text: string;
+};
+
+export function Button(props: Props) {
```
Comment (gerardoh13) [inline react-props/my-app/src/Button.tsx]: Destructured prop

**Review by gerardoh13 — CHANGES_REQUESTED** [review]
Almost perfect, let's use destructured props in these components

**Comment by SheaPone** [comment]
Screenshot 2024-12-05 at 9 19 25 PM

**Review by gerardoh13 — APPROVED** [review]
Looks good!

**Pull request merged**